src/pages/index.js

```diff
@@ -3,26 +3,11 @@ import { graphql } from 'gatsby'
 import PropTypes from 'prop-types'
 import Layout from '../components/layout'
 import Home from '../components/home'
-import Button from '../components/button'
-import { navigate } from 'gatsby'
-import { CgArrowLongRight } from 'react-icons/cg'
 class RootIndex extends React.Component {
```

Comment:
White space here too?

Reply:
You are right. For some reason there was white space but it was showing a timestamp for changes. When I looked at the line numbers, I realised there was no whitespace.